### PR TITLE
feat: deal with register denied error

### PR DESCRIPTION
**Pull request by lucianaDilizio**
Catch denied register error:
To test it: https://cdn.fromdoppler.com/doppler-webapp/int-build648/#/signup

image

### CODE REVIEW

**Comment by andresmoschini** [comment]
Nice!

File: src/services/doppler-legacy-client.doubles.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export class HardcodedDopplerLegacyClient implements DopplerLegacyClient {
     console.log(this.registerUser, model);
     await timeout(1500);
     return { success: true };
+    //return { expectedError: { registerDenied: true } };
```
Comment (cbernat) [inline src/services/doppler-legacy-client.doubles.ts]: @lucianaDilizio comment?

Comment (lucianaDilizio) [reply src/services/doppler-legacy-client.doubles.ts]: This is to test different responses in localhost.

Comment (andresmoschini) [reply src/services/doppler-legacy-client.doubles.ts]: These comments are here to test the scenario easily in the local environment.

Comment (cbernat) [reply src/services/doppler-legacy-client.doubles.ts]: Well I see both of you have the same idea about this. It should be nice in the future perhaps adding a comment stating the use of this line.

Comment (andresmoschini) [reply src/services/doppler-legacy-client.doubles.ts]: Nah, if we all already know, the comment is not necessary :P

**Pull request merged**